### PR TITLE
Release Godot 3.5.3 and publish release notes

### PR DESCRIPTION
**Pull request by akien-mga**

### CODE REVIEW

**Review by mhilbrunner — APPROVED** [review]

File: collections/_article/maintenance-release-godot-3-5-3.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+author: Rémi Verschelde
+image: /storage/blog/covers/maintenance-release-godot-3-5-3.webp
+image_caption_title: Hauma
+image_caption_description: A visual novel by SenAm Games
+date: 2023-09-25 14:00:00
+---
+
+Long time no see, Godot 3.5! While our development focus since [Godot 4.0](/article/godot-4-0-sets-sail/) in March has been to release new 4.x versions at a steady pace (with [Godot 4.1](/article/godot-4-1-is-here/) in July, and Godot 4.2 scheduled for November), we're not forgetting about users who are still using Godot 3.5.
+
+Everyone seemed pretty happy with the [3.5.2 release](/article/maintenance-release-godot-3-5-2/), and it's the one most 3.x games have been published with in the past few months. But there's still a number of issues that are worth addressing in that branch, including notably some improvements for platform support such as targeting Android 13 by default. Also, we're still working on the upcoming Godot 3.6 feature release, which should soon reach the release candidate stage.
```
Comment (coppolaemilio) [inline collections/_article/maintenance-release-godot-3-5-3.md]:
```suggestion
Everyone seemed pretty happy with the [3.5.2 release](/article/maintenance-release-godot-3-5-2/), and it's the one most 3.x games have been published within the past few months. But there are still a number of issues that are worth addressing in that branch, including notably some improvements for platform support such as targeting Android 13 by default. Also, we're still working on the upcoming Godot 3.6 feature release, which should soon reach the release candidate stage.
```

**Review by clayjohn — APPROVED** [review]

File: collections/_article/maintenance-release-godot-3-5-3.md (hunk 1 above)
Comment (adamscott) [inline collections/_article/maintenance-release-godot-3-5-3.md]: This sentence feels weird.
> and it's the one most 3.x games have been published with in the past few months

```suggestion
Everyone seemed pretty happy with the [3.5.2 release](/article/maintenance-release-godot-3-5-2/). For the matter, that release has seen most 3.x games being published with it in the past few months. But there's still a number of issues that are worth addressing in that branch, including notably some improvements for platform support such as targeting Android 13 by default. Also, we're still working on the upcoming Godot 3.6 feature release, which should soon reach the release candidate stage.
```

Comment (YuriSizov) [reply collections/_article/maintenance-release-godot-3-5-3.md]: I don't find it weird. 🤔

**Review by adamscott — APPROVED** [review]
Perfection.

**Review by coppolaemilio — APPROVED** [review]

File: collections/_article/maintenance-release-godot-3-5-3.md (hunk 1 above)
Comment (akien-mga) [reply collections/_article/maintenance-release-godot-3-5-3.md]: It's a bit convoluted I guess.

**Pull request merged**